viz.yaml

```diff
@@ -45,3 +45,48 @@ fetch:
 process:
 visualize:
 publish:
+  -
+    id: headerCSS
```

Comment:
I don't think this is needed, the css needs to be added to resources, but the file is coming from vizlab